test/components/dataset-summary/dataset-summary.spec.js

```diff
@@ -0,0 +1,51 @@
+import DatasetSummary from '../../../src/components/dataset/dataset-summary.vue';
+import Property from '../../util/ds-property-enum';
+import useForm from '../../../src/request-data/form';
+import testData from '../../data';
+import { mount } from '../../util/lifecycle';
+import { mockRouter } from '../../util/router';
+import { testRequestData } from '../../util/request-data';
+
+const getContainer = () => ({
+  router: mockRouter(),
+  requestData: testRequestData([useForm], {
+    formDraftDatasetDiff: testData.formDraftDatasetDiffs.sorted(),
+    formDatasetDiff: testData.formDatasetDiffs.sorted()
+  })
+});
+
+describe('Dataset summary', () => {
+  const theories = [0, 1, 2];
```

Comment:
Any particular reason you want to test all three of these cases? Would 0 and non-zero work? I suppose if you compared 1 vs more datasets, you could check that the pluralization worked. It's fine to leave it as it, it just doesn't seem like the 3rd test case is testing anything different.

Reply:
I want to make sure 0 is not crashing the app